### ISSUE TITLE
Error in pruneKnn(): constraints are inconsistent, no solution!

### ISSUE THREAD

**Issue by pk43429** [dgrun/RaceID3_StemID2_package#5]
Hi,

I am trying to analyse a scRNA-seq data-set using the VarID package and ran into errors with the pruneKnn function:

`res <- pruneKnn(d,large=TRUE,regNB=T,knn=10,alpha=10,no_cores=8, pcaComp = 100)

Error in checkForRemoteErrors(val) : 
  8 nodes produced errors; first error: constraints are inconsistent, no solution!`

What do you think might be the problem?
I played around with the input parameters - large, regNB, alpha, pcaComp - with no luck. 
The same count matrix and SCseq object work fine with RaceID and StemID functions.  
And, to check installation issues, I ran pruneKnn() using a different data-set and it ran ok. 

Thank you for your help.

**Comment by dgrun** [comment]
The command is ok and the input seem ok, too, if it runs with RaceID.
I cannot tell what could have caused the error.
Could you attach or send the data (with gene names scrambled or removed) so I can reproduce the error?

**Comment by pk43429** [comment]
Thank you for your quick response. 
The file size exceeds the upload limit. Can I e-mail you a link to the file?

**Comment by dgrun** [comment]
Sure. Please send the link to gruen@ie-freiburg.mpg.de

**Comment by dgrun** [comment]
I fixed the bug in the new version RaceID v0.1.9. It is uploaded to the repository and will be uploaded to CRAN. However, looking at your data I noticed that these are not UMI counts but read counts instead, right? Both RaceID and VarID only work on UMI count data as they rely on noise models that are not valid for read counts. In such cases RaceID might still work without the outlier identification. RaceID's outlier identification and VarID, however, will likely yield non-sensible results.
